app/views/components/summaryCard.scala.html

```diff
@@ -0,0 +1,47 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+
+@this(
+    govukSummaryList : GovukSummaryList,
+    govukTag: GovukTag
+)
+
+@(
+    summaryListRows: Seq[SummaryListRow],
+    status: String,
+    pointValue: Int,
+    footerLink: String = "",
+    tagStatusClasses: String = "penalty-due-tag" // can maybe use this as a temp DUE tag class govuk-tag--red/penalty-due-tag
+)(implicit messages: Messages)
+
+<Section class="app-summary-card govuk-!-margin-bottom-3">
+    <header class="app-summary-card__header">
+        <h4 class="app-summary-card__title">@messages("summaryCard.header", pointValue)</h4>
```

Comment:
On the prototype they use h3 as the previous heading, however we used a h2 for the previous heading. **So this needs to be** `<h3>`